pkgs/top-level/php-packages.nix

```diff
@@ -506,8 +506,11 @@ lib.makeScope pkgs.newScope (self: with self; {
         configureFlags = [ "--enable-xmlreader" ]
           # Required to build on darwin.
           ++ lib.optional (lib.versionOlder php.version "7.4") [ "--with-libxml-dir=${libxml2.dev}" ]; }
-      { name = "xmlrpc";
+      {
+        name = "xmlrpc";
```

Comment:
Let's move name up just at it was before, that's how we do it for all other places in this file :)

Reply:
Oops, I'll move it back.